index.rst

```diff
@@ -39,4 +39,204 @@
+  including Milky Way dust extinction and a standard atmosphere at
+  airmass=1.2 (calculated from the SED, E(B-V) pair)
+
+* umag_noatm,... -- apparent LSST magnitudes of the star, including Milky Way dust
+  extinction, above the atmosphere.
+
+* newSDSSu, newSDSSg, newSDSSr, newSDSSi, newSDSSz -- the apparent SDSS
+  magnitudes of the star, including Milky Way dust extinction (calculated from
+  the SED, E(B-V) pair)
+
+* sdssu, sdssg, sdssr, sdssi, sdssz, -- the apparent SDSS magnitudes
+  negelecting Milky Way dust extinction
+
+* residual -- the RMS color residual between the input stellar colors
+  and the colors of the best-fit SED and E(B-V) pair (in the input catalog
+  magnitudes measured for that star).
+
+* cx, cy, cz -- the Cartesian coordinates of the star on a hypothetical unit
+  sphere (like htmid, these exist to facilitate spatial searching of the table).
+
+
+Validating the Catalog
+======================
+
+Scripts to generate plots useful for validating this catalog are provided in the
+``software/validation/`` directory of `this github repository <https://github.com/lsst-sims/smtn-006>`_.  Because the catalog is so large, these
+scripts come in groups that work together: some scripts read in the catalog, aggregate
+useful quantities, and output those quantities as text files; other scripts read in those
+text files and use Matplotlib to produce plots.  The useful groups of scripts are:
+
+* ``dec_vs_mag.py`` queries the CatSim database for all stars between -20.0 < RA < 20.0
+  and produces a density plot of mangitude versus Dec in each of the LSST bands.
+
+* ``stellar_density_control_arrays.py`` reads in the original ``.csv.gz`` files and compiles
+  them into HEALPIX maps of number density in 0.5 magnitude bins in the input
+  (u, g, r, i, z, y) bands.  These HEALPIX maps are outputted to text files as simple numpy
+  arrays of the number of stars in each HEALPIXel.  ``stellar_density_get_arrays.py``
+  queries the CatSim database and assembles the stars into HEALPIX maps of number density
+  in 0.5 magnitude bins in each of the LSST bands.  ``stellar_density_comparsions.py`` reads
+  in the text files produced by the first two scripts and, for each magnitude bin, plots
+  the number density in the input catalog next to the number density in the CatSim database.
+  Note: these will be different since, in the input catalog, not every star has every
+  magnitude measured.
+
+* ``validate_magnitudes.py`` loops over all of the stars and compiles the number of stars
+  in 0.1 magnitudes bins in both (input magnitude, magnitude residual) space as well as
+  (magnitude residual, color residual) space.  These grids are outputted as text files.
+  ``plot_magnitude_grids.py`` reads in these text files and produces density plots in both
+  of those parameter spaces, as well as cumulative distributions of stars as a function of
+  magnitude residual with different cuts applied to color residual.
```

Comment:
An Example section at the end would be helpful, with the commands you use to set up the catalog and perform some simple task (maybe run one of the validation scripts), along with the output or snippets of the output if it's large.